test/terraform/mzcompose.py

```diff
@@ -0,0 +1,483 @@
+                        helm_chart_version = content["version"]
+                    assert version.endswith(f", helm chart: {helm_chart_version})")
+
+            c.run_testdrive_files(
+                "array.td",
+                "char-varchar-distinct.td",
+                "char-varchar-joins.td",
+                "char-varchar-multibyte.td",
+                "constants.td",
+                "coordinator-multiplicities.td",
+                "create-views.td",
+                "date_func.td",
+                "decimal-distinct.td",
+                "decimal-join.td",
+                "decimal-order.td",
+                "decimal-overflow.td",
+                "decimal-sum.td",
+                "decimal-zero.td",
+                "delete-using.td",
+                "drop.td",
+                "duplicate-table-names.td",
+                "failpoints.td",
+                "fetch-tail-as-of.td",
+                "fetch-tail-large-diff.td",
+                "fetch-tail-limit-timeout.td",
+                "fetch-tail-timestamp-zero.td",
+                "fetch-timeout.td",
+                "float_sum.td",
+                "get-started.td",
+                "github-11563.td",
+                "github-1947.td",
+                "github-3281.td",
+                "github-5502.td",
+                "github-5774.td",
+                "github-5873.td",
+                "github-5983.td",
+                "github-5984.td",
+                "github-6335.td",
+                "github-6744.td",
+                "github-6950.td",
+                "github-7171.td",
+                "github-7191.td",
+                "github-795.td",
+                "joins.td",
+                "jsonb.td",
+                "list.td",
+                "logging.td",
+                "map.td",
+                "multijoins.td",
+                "numeric-sum.td",
+                "numeric.td",
+                "oid.td",
+                "orms.td",
+                "pg-catalog.td",
+                "runtime-errors.td",
+                "search_path.td",
+                "self-test.td",
+                "string.td",
+                "subquery-scalar-errors.td",
+                "system-functions.td",
+                "test-skip-if.td",
+                "type_char_quoted.td",
+                "version.td",
+            )
+    finally:
+        if port_forward_process:
+            os.killpg(os.getpgid(port_forward_process.pid), signal.SIGTERM)
+
+        if args.cleanup:
+            print("--- Cleaning up")
+            if materialize_environment:
+                run_ignore_error(
+                    ["kubectl", "delete", "-f", "-"],
+                    cwd=path,
+                    stdin=yaml.dump(materialize_environment).encode(),
+                )
+            run_ignore_error(
+                ["helm", "uninstall", "materialize-operator"],
+                cwd=path,
+            )
+            # TODO: This is hacky and should not be necessary
```

Comment:
yeah, patching out the finalizer shouldn't be necessary if you move the `kubectl delete` of the environment up above the `helm uninstall` call - we shouldn't be fiddling with finalizers by hand like this